facade/src/test/java/org/jboss/pnc/facade/providers/BuildProviderImplTest.java

```diff
@@ -299,6 +300,44 @@ public void testFilterRunningBuildsByBuildConfigName() {
         assertEquals(BuildMapper.idMapper.toDto(givenBT.getId()), it.next().getId());
     }
 
+    @Test
+    public void testFilterLikeRunningBuildsByBuildConfigName() {
+        // Given
+        mockBuildRecord();
+        mockBuildTask();
+
+        String givenBcName = "VeryLongAndComplicatedBcName";
```

Comment:
You missed an opportunity to name it "VeryLongAndHardToRememberBcName" :smile:

Reply:
:-)